pkg/volume/util/volumepathhandler/volume_path_handler.go

```diff
@@ -31,9 +31,7 @@ import (
 
 const (
 	losetupPath           = "losetup"
-	statPath              = "stat"
 	ErrDeviceNotFound     = "device not found"
-	ErrDeviceNotSupported = "device not supported"
```

Comment:
Need to fix indents in this file to pass pull-kubernetes-verify test job.
Please update this file with running `./hack/update-gofmt.sh`.

Reply:
Thanks, I will fix it